website/docs/components/alert/05--accessibility.md

```diff
@@ -5,15 +5,21 @@ component: alert
 section: accessibility
 ---
 
-This component is conditionally conformant. That is, it is conformant when there are no interactive elements present inside of the alert. There is future work planned to make this component WCAG conformant by adding support for the correct ARIA roles when interactive elements are contained within the alert.
+This component is conditionally conformant.
+
+It is fully conformant when there are no interactive elements present inside of the alert. 
+
+There is future work planned to improve this component's WCAG conformance by adding support for the correct ARIA roles when interactive elements are contained within the alert.
 
 #### Applicable WCAG Success Criteria (Reference)
 
-This section is for reference only. This component intends to conform to the following WCAG success criteria:
+This section is for education and reference. This component intends to conform to the following WCAG success criteria:
+
+<Doc::WcagList @criteriaList={{array '1.3.1' '1.3.2' '1.4.1' '1.4.3' '1.4.10' '1.4.11' '1.4.12' '2.1.1' '2.1.2' '2.2.1' '2.5.3' '4.1.1' '4.1.2' '4.1.3' }} />
```

Comment:
@alex-ju @KristinLBradley what do you think about this? Would this be relatively easy for you to use? Is there something else that would make it easier for you?

Reply:
This seems pretty straight forward to use for me.